tests/process_manager/views/test_page_views.py

```diff
@@ -0,0 +1,101 @@
+from http import HTTPStatus
+from uuid import uuid4
+
+from django.urls import reverse
+from pytest_django.asserts import assertContains, assertTemplateUsed
+
+from ...utils import LoginRequiredTest, PermissionRequiredTest
+
+
+class TestIndexView(LoginRequiredTest):
+    """Tests for the index view."""
+
+    endpoint = reverse("process_manager:index")
+
+    def test_authenticated(self, auth_client):
+        """Test the index view for an authenticated user."""
+        with assertTemplateUsed(template_name="process_manager/index.html"):
+            response = auth_client.get(self.endpoint)
+        assert response.status_code == HTTPStatus.OK
+
+    def test_admin(self, admin_client):
+        """Test the index view for an admin user."""
+        with assertTemplateUsed(template_name="process_manager/index.html"):
+            response = admin_client.get(self.endpoint)
+        assert response.status_code == HTTPStatus.OK
+        assertContains(response, "Boot</a>")
```

Comment:
This test doesn't add anything, if you replace `admin_client` with `auth_client` it will still pass.

Reply:
I agree. Removed.